components/Banner/Banner.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable camelcase */
+import { Icon } from "@equinor/eds-core-react"
+import { warning_filled } from "@equinor/eds-icons"
+import { FunctionComponent, ReactNode } from "react"
+
+type Props = {
+  variant: string
+  children: ReactNode
+}
+
+export const Banner: FunctionComponent<Props> = ({ variant, children }) => {
+  const classNames = [
+    "banner",
+    variant === "danger" ? "banner--danger" : "banner--warning",
+  ]
+    .join(" ")
+
+  return (
+    <div className={classNames}>
```

Comment:
Pay attention to the centering and alignment of the icon and text in the Figma sketches. The icon should not be centered along the y-axis if we have several paragraphs of text, but if we have only one line of text both the icon and text should be centered. Now it is a little bit off.
![Skjermbilde 2022-08-03 kl  10 01 16](https://user-images.githubusercontent.com/1568252/182556216-85055e51-88cd-4a81-b940-5921af83450a.png)

Reply:
Need to have discussion on this one . I tried some things but still haven't got perfect results.

Reply:
We have discussed this and action points
- Remove the magic whitespace because of the inline element by adding `line-height: 0` to the wrapping icon div
- Add a flex-box and an align-items: center to align the text
- Test, as we just tested this quick and dirty 😅